docs/src/ComponentsPage.js

```diff
@@ -574,6 +574,24 @@ const ComponentsPage = React.createClass({
                   <code>getValue()</code> will not work when used this way.</p>
                   <ReactPlayground codeText={Samples.InputWrapper} />
                 </div>
+
+                {/* Menuitem */}
+                <div className='bs-docs-section'>
+                  <h1 id='menu-item' className='page-header'>MenuItem</h1>
```

Comment:
For consistency this title should probably follow the same format as [List group](http://localhost:4000/components.html#listgroup). Note an English friendly spaced variant with `small` component names to its side.